store.go

```diff
@@ -63,3 +66,13 @@ func (s *s3store) Delete(appId string, backupId string) error {
 func (*s3store) pathFor(appId string, backupId string) string {
 	return fmt.Sprintf("pgbackups/%s/%s.backup", appId, backupId)
 }
+
+func setRegion(region string) string {
```

Comment:
let's rename this to "getRegion" for clarity, and arg name should be "regionName" or maybe just "name" for consistency